python/cxx/export.cpp

```diff
@@ -30,22 +32,10 @@ void python::common::export_all(py::module& module) {
 
 PYBIND11_MODULE(opmcommon_python, module) {
     python::common::export_all(module);
-}
 
-void python::common::export_all_opm_embedded(py::module& module) {
-    python::common::export_all(module);
+    // These attributes are placeholders, they get set to the actual EclipseState, Schedule and SummaryState in PyRunModule.cpp
     module.attr("current_ecl_state") = std::make_shared<EclipseState>();
```

Comment:
@hakonhagland: I changed this, such that the attributes are not set on a submodule anymore, like this, the stub file can be used as it is and does not need to be adjusted anymore. It only needs to be renamed and put in the folder at "python.analysis.stubPath".
I've also added a comment about that in 683543e288e28005ae0d08b529281d8e2c75647f.

Reply:
I guess these attributes only make sense in the sub module `opm.embedded` ? Then it might be better to keep them in the sub module instead of "polluting" `opmcommon_python` with variables to simplify the generation of the type info files?

Reply:
@hakonhagland: Ok, I've reverted the change and added the stub file to github as well.